src/main/java/com/techiteasy/demo/models/Television.java

```diff
@@ -38,35 +38,36 @@ public class Television {
     //Data worden opgeslagen als DATE of TIMESTAMP types in de database
     //Enums worden opgeslagen als VARCHAR of ENUM type in de database
 
-    //Default constructor - dit is handig wanneer je een object wilt aanmaken zonder onmiddellijk waarden toe te wijzen aan de velden.
-    public Television() {
+    //OneToOne-relatie met RemoteController
+    //De 'CascadeType.All' zorgt ervoor dat alle cascade-operaties (zoals persist, merge, remove) worden toegepast op de gekoppelde Remotecontroller wanneer er iets gebeurt met de Television.
+    @OneToOne(cascade = CascadeType.ALL)
+    @JoinColumn(name = "remote_controller_id", referencedColumnName = "id")
+    private RemoteController remoteController;
+
+    //OneToMany-relatie met CIModule
+    @OneToMany(mappedBy = "television", cascade = CascadeType.ALL)
```

Comment:
Television kan nu niet de owner zijn van de relatie, omdat een television meerdere cimodules kan hebben. In de opdracht beschrijving stond het echter andersom, "Hierbij is het nodig dat meerdere tv's één ci-module kunnen hebben" (oftwel, een cimodule heeft meerdere televisions)